lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex

```diff
@@ -85,38 +84,56 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackSender do
         _other -> []
       end)
 
-    actions = actions ++ [start_timer: {:check_encoding_statuses, Time.seconds(1)}]
+    # start timer only for simulcast tracks
+    actions =
+      actions ++
+        if state.track.simulcast_encodings != [] do
+          [start_timer: {:check_encoding_statuses, Time.seconds(1)}]
+        else
+          []
+        end
```

Comment:
Maybe a little less indentation?
```suggestion
      if state.track.simulcast_encodings != [] do
        actions ++ [start_timer: {:check_encoding_statuses, Time.seconds(1)}]
      else
        actions
      end
```